gateway-ha/src/test/java/io/trino/gateway/ha/router/TestTrinoQueryProperties.java

```diff
@@ -91,4 +100,185 @@ void testJsonCreatorWithEmptyProperties()
         assertThat(deserializedTrinoQueryProperties.isQueryParsingSuccessful()).isEqualTo(trinoQueryProperties.isQueryParsingSuccessful());
         assertThat(deserializedTrinoQueryProperties.getErrorMessage()).isEqualTo(trinoQueryProperties.getErrorMessage());
     }
+
```

Comment:
Could you please add tests involving multiple catalogs and schemas, may be with join statements? to test the catalogs and schema arrays.

Reply:
Nice!